Yoast/javascript
# Add Swedish external stemmer

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds and refactors an external stemmer for Swedish
* [Yoast SEO Premium] Adds word forms support for Swedish keyphrase recognition.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` and make sure that all specs pass
* In content analysis app:
    * Turn the morphology feature on and add minimum 150 words to the text field
    * Set the locale to "sv"
    * Use `barnslig` as the focus keyphrase 
    * Add these other forms of the word in the text field `barnsliga, barnslighet, barnsligheten, barnsligheter, barnsligt` and make sure that all these forms are recognized.

## QA test instruction
* In your testing environment, set the language to Swedish
* Make sure to test this functionality in premium
* Add text minimum 150 words
* Use `bedrag` as the focus keyphrase 
* Add these other forms of the word in the text field `bedraga, bedragare, bedragen, bedrager` and make sure that all these forms are recognized.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-558
